files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_B.ifc

```diff
@@ -0,0 +1,38 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [Alignment-basedReferenceView]'), '2;1');
+FILE_NAME ('ACCA_IfcTunnel_Sprint_1_2_Georef_B.ifc', '2023-02-16T14:56:48', ('Alfonso D\X2\0027\X0\Urso', 'alfonso.durso@acca.it'), ('ACCA software S.p.A.'), 'ACCA_IFC.dll Version 500', 'ACCA usBIM.infra', $);
+FILE_SCHEMA (('IFC4X4'));
+ENDSEC;
+DATA;
+#1 = IFCOWNERHISTORY(#2, #6, $, .NOCHANGE., $, $, $, 1676559408);
+#2 = IFCPERSONANDORGANIZATION(#3, #4, $);
+#3 = IFCPERSON($, 'D\X2\0027\X0\Urso', 'Alfonso', $, $, $, $, $);
+#4 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#5 = IFCORGANIZATION($, 'ACCA software', $, $, $);
+#6 = IFCAPPLICATION(#5, '200', 'ACCA usBIM.infra', 'ACCA usBIM.infra');
+#7 = IFCPROJECT('33UpoFiCf06eZR8EQGZH2R', #1, 'IfcTunnel example', $, $, $, $, (#13), #8);
+#8 = IFCUNITASSIGNMENT((#9, #10, #11, #12));
+#9 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#10 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#11 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#12 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#13 = IFCGEOMETRICREPRESENTATIONCONTEXT('3D', 'Model', 3, 1.E-4, #14, #16);
+#14 = IFCAXIS2PLACEMENT3D(#15, $, $);
+#15 = IFCCARTESIANPOINT((0., 0., 0.));
+#16 = IFCDIRECTION((0., 1.));
+#17 = IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Body', 'Model', *, *, *, *, #13, $, .MODEL_VIEW., $);
+#18 = IFCSITE('0393X_CUP0ZPyKwIINUUYL', #1, 'Default Site', $, $, #19, $, $, .ELEMENT., (0, 0, 0), (0, 0, 0), 0., $, $);
+#19 = IFCLOCALPLACEMENT($, #22);
+#20 = IFCDIRECTION((0., 0., 1.));
+#21 = IFCDIRECTION((1., 0., 0.));
+#22 = IFCAXIS2PLACEMENT3D(#15, #20, #21);
+#23 = IFCRELAGGREGATES('1SfvBitqn5vv1js13bQycY', #1, 'Project Container', 'Project to Spatial Element', #7, (#18));
+#24 = IFCCLASSIFICATION('Default Source', 'Default Edition', '2000-01-01', 'Default Name', $, $, $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('3XlHkb_JX3RBSwRgdO28$y', #1, $, $, (#7), #24);
+#26 = IFCGROUP('12aW0JZoP5v8qiYDw1lDKu', #1, 'A group', $, $);
+#27 = IFCRELDECLARES('0QhXIT61H0CxB10IWPGV9e', #1, 'Project Declaration Relationship', $, #7, (#26));
+#28 = IFCRIGIDOPERATION(#13, #29, IFCPLANEANGLEMEASURE(0.), IFCPLANEANGLEMEASURE(0.), 0.);
+#29 = IFCGEOGRAPHICCRS('EPSG:3065', $, $, $, $, $);

```

Comment:
`'EPSG:3065'` is not a geographic CRS, but a projected CRS ([source](https://epsg.io/3065)). Please double-check.

Reply:
Fixed